src/MusicalScore/ScoreIO/InstrumentReader.ts

```diff
@@ -385,11 +375,20 @@ export class InstrumentReader {
             this.currentStaffEntry.Timestamp.Equals(new Fraction(0, 1)) && !this.currentStaffEntry.hasNotes()
           );
           this.saveAbstractInstructionList(this.instrument.Staves.length, beginOfMeasure);
-          if (this.openChordSymbolContainers.length !== 0) {
-            this.currentStaffEntry.ChordContainers = this.openChordSymbolContainers;
-            // TODO handle multiple chords on one note/staffentry
-            this.openChordSymbolContainers = [];
-          }
+          //TODO: this if block handles harmony/chords on the next note/staffentry element, so it assumes that a
+          //harmony is given before the staff entry, but when a harmony is given after a staff entry element with a backup node
+          //it is put on the next note/staffentry and the last chord item is never parsed at all.
+          //this means
+          //Ik moet eerst beter begrijpen wanneer currentStaffEntry geset wordt.
+          // if (this.openChordSymbolContainers.length !== 0) {
+          //   console.log("hier is iets gevonden in de chordsymbol container");
+          //   this.currentStaffEntry.ChordContainers = this.openChordSymbolContainers;
+          //   console.log("currentStaffENtry.ChordContainers:");
+          //   console.debug(this.currentStaffEntry);
+          //   console.debug(currentFraction);
+          //   // TODO handle multiple chords on one note/staffentry
+          //   this.openChordSymbolContainers = [];
+          // }
```

Comment:
I commented line 390 and the if around it, because the included tslint complained about the openChordSymbolContainers property not being used anywhere. I also studied this file for a long time to know how it works and concluded that it's approach is wrong, because the chords were buffered until the next iteration of a xmlNode. So in my opinion it should go. But if you think otherwise, please let me know, then I'll try to help with a solution for that.